src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -140,15 +142,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String messageContent = request.getParameter("message");
 
-    // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
-
```

Comment:
I am not sure if I completely understand, but I think I do. I agree. We would only change this line to String cleanedMessageContent = Jsoup.clean (messageContent, Whitelist.basic()) so it doesn't remove all the hlml tags, since we want those to be passed to the jsp file. Thanks!